knoten/csm.py

```diff
@@ -73,6 +96,35 @@ def create_camera(label, url='http://pfeffer.wr.usgs.gov/api/1.0/pds/'):
     if plugin.canModelBeConstructedFromISD(isd, model_name):
         model = plugin.constructModelFromISD(isd, model_name)
         return model
+    
+def get_state(sensor, image_pt):
```

Comment:
+1, could we get a type check on the sensor?

There may also be some fancy argument registration we can do for the sensor parameter